src/config/index.js

```diff
@@ -1,14 +1,24 @@
-export const NETWORK = process.env.NET || "ropsten";
+const IS_DEVELOPMENT = process.env.NODE_ENV === "development" || process.env.NODE_ENV === "test";
 
-export const IS_MAINNET = NETWORK === "mainnet";
+const NETWORK = process.env.NET || "ropsten";
+const IS_MAINNET = NETWORK === "mainnet";
+const NETWORK_NAME = IS_MAINNET ? "homestead" : NETWORK;
+const ETHERSCAN_SUBDOMAIN = IS_MAINNET ? "" : `${NETWORK_NAME}.`;
 
-export const INFURA_PROJECT_ID = "1f1ff2b3fca04f8d99f67d465c59e4ef";
-export const INFURA_API_KEY = process.env.INFURA_API_KEY || "bb46da3f80e040e8ab73c0a9ff365d18";
-
-export const NETWORK_ID = IS_MAINNET ? "1" : "3";
-export const NETWORK_NAME = IS_MAINNET ? "homestead" : NETWORK;
-
-export const ETHERSCAN_BASE_URL = `https://${IS_MAINNET ? "" : NETWORK_NAME + "."}etherscan.io/`;
-export const MAGIC_API_KEY = process.env.MAGIC_API_KEY || "pk_test_AB1F885AF848182E";
-
-export const isDevelopment = process.env.NODE_ENV === "development" || process.env.NODE_ENV === "test";
+module.exports = {
```

Comment:
export default instead?

Reply:
like can't, require and export don't play well together